apps/web/src/pages/CandidateInfoPage/CandidateInfoPage.tsx

```diff
@@ -0,0 +1,23 @@
+import { useRoute } from 'wouter';
+
+import { useFetchIntern } from '../../api/useFetchIntern';
+import CandidateInfo from '../../components/CandidateInfo';
+import { Path } from '../../constants/paths';
+
+const CandidateInfoPage = () => {
+  const [, params] = useRoute(Path.Candidate);
+  const internId: string | undefined = params?.internId;
+  const { data: intern, isFetching } = useFetchIntern(internId as string);
+
+  if (isFetching) {
+    return <div>Učitavanje...</div>;
+  }
+
```

Comment:
fali handling ako ne postoji intern, mozes koristit isError iz react-queryja za to:

```
  if (isError) {
    return <div>Dogodila se greška</div>
  }
```